### PR TITLE
ensure an nginx user exists

### PR DESCRIPTION
**Pull request by alfredodeza**
And remove mongodb, not sure how I managed to copy/pasta'd there

### CODE REVIEW

File: deploy/playbooks/roles/common/tasks/nginx.yml [hunk 1]
```diff
@@ -3,6 +3,11 @@
   file: path=/etc/nginx/sites-available state=directory
   sudo: true
 
+# Add the user 'johnd' with a specific uid and a primary group of 'admin'
```
Comment (andrewschoen) [inline deploy/playbooks/roles/common/tasks/nginx.yml]: who's johnd?

Comment (alfredodeza) [inline deploy/playbooks/roles/common/tasks/nginx.yml]: how is it possible you don't know him

Comment (andrewschoen) [inline deploy/playbooks/roles/common/tasks/nginx.yml]: I must know him by his alias, 'nginx'. :)

Comment (alfredodeza) [inline deploy/playbooks/roles/common/tasks/nginx.yml]: it is another alias for @vasukulkarni

**Pull request merged**